### PR TITLE
For issue 77 - DRY issue in template_sampling module

### PR DESCRIPTION
**Pull request by kamuda1**
Adds new function in template_sampling to remove duplicate code. Tests for new function and other template_sampling functions are included.

### CODE REVIEW

File: annsa/template_sampling.py [hunk 1]
```diff
@@ -100,6 +104,54 @@ def apply_LLD(spectrum, LLD=10):
...
 
 
+def construct_spectrum(spectral_template,
+                       calibration=[0, 1, 0],
+                       LLD=10,
+                       spectrum_counts=60,
+                       output_len=1024,
+                       ):
+    """
+    This function manipulates a spectral template.
```
Comment (katyhuff) [inline annsa/template_sampling.py]: can you be more specific?

File: annsa/tests/test_templatesampling.py [hunk 2]
```diff
@@ -0,0 +1,44 @@
+from annsa.template_sampling import (rebin_spectrum,
+                                     apply_LLD,
+                                     construct_spectrum,)
+import numpy as np
+
+# define lamba = 1000
```
Comment (katyhuff) [inline annsa/tests/test_templatesampling.py]: this should go in a setup fixture.

**Review by katyhuff — CHANGES_REQUESTED** [review]

**Comment by katyhuff** [comment]
Flake 8 : 

```bash
/home/circleci/annsa/annsa/template_sampling.py:215:13: E123 closing bracket does not match indentation of opening bracket's line
/home/circleci/annsa/annsa/template_sampling.py:228:13: E123 closing bracket does not match indentation of opening bracket's line
/home/circleci/annsa/annsa/template_sampling.py:240:13: E126 continuation line over-indented for hanging indent
/home/circleci/annsa/annsa/template_sampling.py:244:13: E123 closing bracket does not match indentation of opening bracket's line
```

**Review by samgdotson — APPROVED** [review]
I think these changes make sense. The tests are good.

**Review by kamuda1 — COMMENTED** [review]
All changes have been made.

**Comment by kamuda1** [comment]
@katyhuff The changes have been made, I think this is good to merge.

**Review by katyhuff — APPROVED** [review]

**Pull request merged**